charts/mod-circulation-bff/values.yaml

```diff
@@ -159,6 +159,12 @@ integrations:
     url: http://okapi:9130
     existingSecret: ""
 
+  systemuser:
+    enabled: false
+    name: mod-circulation-bff
```

Comment:
the same should be presented in TF config part & config files in resources folder. kfyi please.

Reply:
Required code is already there:

https://github.com/folio-org/pipelines-shared-library/blob/7a269f419bbefa89704c23520626cea0a2d15731/terraform/rancher/project/locals.tf#L15
https://github.com/folio-org/pipelines-shared-library/blob/7a269f419bbefa89704c23520626cea0a2d15731/resources/helm/development.yaml#L732
https://github.com/folio-org/pipelines-shared-library/blob/7a269f419bbefa89704c23520626cea0a2d15731/resources/helm/testing.yaml#L829
https://github.com/folio-org/pipelines-shared-library/blob/7a269f419bbefa89704c23520626cea0a2d15731/resources/helm/performance.yaml#L829